File: src/game/level/world2/World2Level3.java
package game.level.world2;

import game.entity.enemies.boss.Boss;
import game.entity.enemies.npcs.FlyingEnemy;
import game.entity.enemies.npcs.YellowBall;
import game.entity.enemies.npcs.Zombie;
import game.entity.models.Enemy;
import game.level.Level;
import game.level.world1.World1Level;

public class World2Level3 extends World2Level {
    @Override
    public int getLevelId() {
        return 3;
    }

    @Override
    public int startEnemiesCount() {
        return 15;
    }

    @Override
    public Class<? extends Enemy>[] availableEnemies() {
        return new Class[]{
                Zombie.class,
                FlyingEnemy.class
        };
    }

    @Override
    public Class<? extends Level> getNextLevel() {
        return World2Level4.class;
    }
}

File: src/game/level/world2/World2Level3.kt
package game.level.world2

import game.entity.enemies.npcs.FastEnemy
import game.entity.enemies.npcs.TankEnemy
import game.entity.models.Enemy
import game.level.Level

class World2Level3 : World2Level() {
    override val levelId: Int
        get() {
            return 3
        }

    override val startEnemiesCount: Int
        get() {
            return 13
        }

    override val availableEnemies: Array<Class<out Enemy>>
        get() {
            return arrayOf(
                    TankEnemy::class.java,
                    FastEnemy::class.java)
        }

    override val nextLevel: Class<out Level?>
        get() {
            return World2Level4::class.java
        }
}